### PR TITLE
landing v.2 iteration + terms & guideline pages v.2

### PR DESCRIPTION
**Pull request by tarafanlin**
- restructure terms & guideline pages
- adding page content menu
- responsive for mobile & tablet

### CODE REVIEW

File: pages/terms-new.js [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+  }
+  return (
+    <div css={STYLES_COPY_EMAIL}>
+      <a onClick={copyToClipboard}>abuse@filecoin.io</a>
+      {copySuccess}
+    </div>
+  );
+}
+
+export default class GuidelinesPage extends React.Component {
```
Comment (jimmylee) [inline pages/terms-new.js:131]: Nit Bad copy & paste?

File: pages/guidelines-new.js [hunk 2]
```diff
@@ -0,0 +1,383 @@
...
+  :visited {
+    color: ${Constants.system.slate};
+  }
+`;
+
+const STYLES_COPY_EMAIL = css`
+  display: inline;
+`;
+
+export const getServerSideProps = async (context) => {
```
Comment (jimmylee) [inline pages/guidelines-new.js:112]: Why do you need `getServerSideProps`? is anything coming from the server?

**Review by jimmylee — CHANGES_REQUESTED** [review]

File: pages/terms-new.js (hunk 1 above)
Comment (tarafanlin) [reply pages/terms-new.js:131]: yep  😛

File: pages/guidelines-new.js (hunk 2 above)
Comment (tarafanlin) [reply pages/guidelines-new.js:112]: actually I'm not sure,,,,,I saw other pages have this so just put it there.

Comment (jimmylee) [reply pages/guidelines-new.js:112]: This is only necessary if you need to pass props from the server to the client. I don't think you need this for marketing pages so you can delete it.

**Comment by jimmylee** [comment]
LGTM

**Pull request merged**